app/views/videos/index.html.haml

```diff
@@ -3,7 +3,7 @@
     %header
       = link_to category_path(category) do
         %h3= category.name.upcase
-      .videos.row
+      .video.row
```

Comment:
Thanks..i forget i was playing with this. The reason why , I was going thru the bootstrap/haml and I could not find the .video or .videos class.

Its a question I was going to ask, but forget..do you know where the .videos definition is?